scripts/wikipedia.js

```diff
@@ -8,14 +8,38 @@ function addCitations () {
     for (let book of books) {
       let isbn = getISBNFromCitation(book)
       let id = getIdentifier(data[isbn])
+      let metadata = getMetadata(data[isbn])
       if (id) {
-        let link = createLinkToArchive(id)
-        book.parentElement.append(link)
+        let link = createLinkToArchive(id, metadata)
+        // link.attr(addTooltip(metadata))
+        let tooltip = addTooltip(metadata).append(link).tooltip({
+          animated: 'fade',
+          placement: 'auto top',
+          html: true,
+          delay: 500
+        })
+        book.parentElement.append(tooltip[0])
       }
     }
   })
 }
-
+function addTooltip (metadata) {
+  return $('<a>').attr({
+    'data-toggle': 'tooltip',
+    'title': createTooltipWindow(metadata)[0].outerHTML
+  })
+}
+function createTooltipWindow (metadata) {
+  let text_elements = $('<div>').attr({ 'class': 'text_elements' }).append(
+    $('<p>').append($('<strong>').text(metadata.title)),
+    $('<p>').css('color', '#c3c3c3').text(metadata.author)
```

Comment:
Isn't it better to use a CSS class here? Please try bootstrap `.text-muted` if its OK with you
https://www.w3schools.com/bootstrap/bootstrap_ref_css_helpers.asp

Reply:
I didn't know about this bootstrap class.  This is super useful, thanks!